CHANGELOG.md

```diff
@@ -1477,14 +1477,14 @@ Swift 2.0
   For example:
 
   ```swift
-  @available(iOS 8.0, OSX 10.10, *)
+  @available(iOS 8.0, macOS 10.10, *)
   func startUserActivity() -> NSUserActivity {
     ...
   }
   ```
 
   This code fragment indicates that the `startUserActivity()` method is
-  available on iOS 8.0+, on OS X v10.10+, and on all versions of any other
+  available on iOS 8.0+, on macOS v10.10+, and on all versions of any other
```

Comment:
I wouldn't have bothered changing the changelog, especially since 10.10 really was called "OS X" at the time. Either way is fine, though.